ci_config.json

```diff
@@ -32,6 +32,18 @@
       "glib:tests=false"
     ]
   },
+  "imgui": {
+    "_comment": ["- Disable sdl_renderer in CI as version 2.0.17+ is required."],
```

Comment:
It's a bit unfortunate that this apparently.needs the development version of sdl2?

Anyway this should probably be a version: kwarg on the dependency lookup.

Reply:
Also yes, adding the version kwarg to the first sdl2 lookup will cause the subproject to not be a valid satisfier
 But the second sdl2 lookup will work fine as it doesn't have a version restriction, so sdl2 will be enabled but not sdl_renderer.

Reply:
> It's a bit unfortunate that this apparently needs the development version of sdl2?

Yes a bit unlucky, but better to implement now  (https://github.com/ocornut/imgui/blob/55d35d8387c15bf0cfd71861df67af8cfbda7456/backends/imgui_impl_sdlrenderer.cpp#L1-L2).

> Anyway this should probably be a version: kwarg on the dependency lookup.

Good point, done!